### ISSUE TITLE
[CRASH] Opensips 3.4.1 using launch() on avp_db_query

### ISSUE THREAD

**Issue by Hyleoss** [OpenSIPS/opensips#3210]
**OpenSIPS version you are running**

```
version: opensips 3.4.1 (x86_64/linux)
flags: STATS: On, DISABLE_NAGLE, USE_MCAST, SHM_MMAP, PKG_MALLOC, Q_MALLOC, F_MALLOC, HP_MALLOC, DBG_MALLOC, FAST_LOCK-ADAPTIVE_WAIT
ADAPTIVE_WAIT_LOOPS=1024, MAX_RECV_BUFFER_SIZE 262144, MAX_LISTEN 16, MAX_URI_SIZE 1024, BUF_SIZE 65535
poll method support: poll, epoll, sigio_rt, select.
git revision: 3a7a2bdfb
main.c compiled on  with gcc 10
```

**Crash Core Dump**
https://pastebin.com/ckVtFFU1

**Describe the traffic that generated the bug**
Classic INVITE with use of avp_db_query with launch()

**To Reproduce**
1. Use launch(avp_db_query("INSERT INTO XXXX"); in opensips.cfg when incoming call
2. Start OpenSIPS
3. make a test call
4. Crash

**Relevant System Logs**
```
Sep 26 19:32:27 OSF-IT-V-OPENSIPS-LAB-1 /usr/sbin/opensips[3088462]: DBG:db_postgres:db_postgres_connect: PSQL_PARAM connect_timeout=5
Sep 26 19:32:27 OSF-IT-V-OPENSIPS-LAB-1 /usr/sbin/opensips[3088462]: DBG:db_postgres:db_postgres_connect: PSQL_PARAM host=localhost
Sep 26 19:32:27 OSF-IT-V-OPENSIPS-LAB-1 /usr/sbin/opensips[3088462]: DBG:db_postgres:db_postgres_connect: PSQL_PARAM dbname=XXX
Sep 26 19:32:27 OSF-IT-V-OPENSIPS-LAB-1 /usr/sbin/opensips[3088462]: DBG:core:db_init_async: >>    1/10 transfers: (5 - 0x7f68a4297a58)
Sep 26 19:32:27 OSF-IT-V-OPENSIPS-LAB-1 /usr/sbin/opensips[3088462]: DBG:db_postgres:db_postgres_submit_async_query: 0x7f68a4294318 PQsendQuery(INSERT INTO XXX VALUES (default,'+XXXXXXXXXXX','+XXXXXXXXXXX','+XXXXXXXXXXX',default,''))
Sep 26 19:32:27 OSF-IT-V-OPENSIPS-LAB-1 /usr/sbin/opensips[3088462]: CRITICAL:core:sig_usr: segfault in process pid: 3088462, id: 12
Sep 26 19:32:27 OSF-IT-V-OPENSIPS-LAB-1 /usr/sbin/opensips[3088462]: DBG:core:restore_segv_handler: restoring SIGSEGV handler...
 ```

**OS/environment information**
 - Operating System: `Debian 11`
 - OpenSIPS installation: `package from repo`
 - other relevant information:

**Additional context**
The crash doesn't appear when using avp_db_query() without launch().

I updated from OpenSIPS 3.3.6 to 3.4.1, and didn't had this crash in 3.3

**Comment by bogdan-iancu** [comment]
@Hyleoss, a fix is available see 26d6d95ffde2bfcdcc67f134c3056dc893c0aad9 - if you could confirm it, it would be great.

**Comment by Hyleoss** [comment]
Problem is resolved with this fix, thank @bogdan-iancu
